.babelrc.js

```diff
@@ -7,4 +7,17 @@ const presets = [
   ],
 ];
 
-module.exports = { presets };
+const testPresets = [
+  [
+    "@babel/react",
```

Comment:
Why `@babel/react`? 

In Jest docs they recommend to use `@babel/preset`:

https://jestjs.io/docs/en/getting-started#using-babel

Reply:
`@babel/preset-env` is for advanced ES6+ transpilation `@babel/react` does the same for JSX, etc. (react-test-renderer), see: https://jestjs.io/docs/en/tutorial-react